Possibly wrong include path in baseclient.h

Hello,
I'm having troubles compiling PHD2 on openSUSE because `indimacros.h` cannot be found (but it's preset in the system).

```
[   56s] In file included from /home/abuild/rpmbuild/BUILD/phd2-2.6.10/phdindiclient.h:37,
[   56s]                  from /home/abuild/rpmbuild/BUILD/phd2-2.6.10/config_indi.h:43,
[   56s]                  from /home/abuild/rpmbuild/BUILD/phd2-2.6.10/stepguider_sbigao_indi.cpp:44:
[   56s] /usr/include/libindi/baseclient.h:27:10: fatal error: indimacros.h: No such file or directory
[   56s]    27 | #include <indimacros.h>
```

Looking at the file `libs/indibase/baseclient.h`, it seems that it's including `indimacros.h` while, AFAICS, it should be `libindi/indimacros.h`.

https://github.com/indilib/indi/blob/6337561bbcd15e41efeceae1e1fb983758ee3f6e/libs/indibase/baseclient.h#L27

What do you think?

--- a/libs/indibase/baseclient.h
+++ b/libs/indibase/baseclient.h
@@ -20,11 +20,11 @@
 
 #include "indiapi.h"
 #include "indibase.h"
+#include "indimacros.h"
 
 #include <string>
 #include <vector>
 
-#include <indimacros.h>
 #include <memory>
 
 // #define MAXRBUF 2048 // #PS: defined in indibase.h

